template/base/src/env/schema.mjs

```diff
@@ -0,0 +1,28 @@
+// @ts-check
+import { z } from "zod";
+
+/**
+ * Specify your server-side environment variables schema here.
+ * This way you can ensure the app isn't built with invalid env vars.
+ */
+export const serverSchema = z.object({
+  // FOO: z.string(),
+});
+
+/**
+ * Specify your client-side environment variables schema here.
+ * This way you can ensure the app isn't built with invalid env vars.
+ * To expose them to the client, prefix them with `NEXT_PUBLIC_`.
+ */
+export const clientSchema = z.object({
+  // NEXT_PUBLIC_BAR: z.string(),
+});
+
+/**
+ * You can't destruct `process.env` as a regular object, so you have to do
+ * it manually here. This is because Next.js evaluates this at build time,
+ * and only used environment variables are included in the build.
+ */
+export const clientEnv = {
```

Comment:
Since this isn't schema config, would it be appropriate to include it in `client.mjs` instead?

Reply:
Then you would have to change between files which seems messy. As it is now you dont even have to look st client/server.mjs. Atleast i prefer that